src/api/ucc.h

```diff
@@ -225,9 +225,9 @@ typedef enum {
  *
  */
 typedef enum {
-    UCC_THREAD_SINGLE       = 0, /*!< Single-threaded library model */
-    UCC_THREAD_FUNNELED     = 1, /*!< Funnel thread model */
-    UCC_THREAD_MULTIPLE     = 2  /*!< Multithread library model */
+    UCC_THREAD_SINGLE       = UCC_BIT(1), /*!< Single-threaded library model */
+    UCC_THREAD_FUNNELED     = UCC_BIT(2), /*!< Funnel thread model */
+    UCC_THREAD_MULTIPLE     = UCC_BIT(3)  /*!< Multithread library model */
 } ucc_thread_mode_t;
```

Comment:
Why? Library saying UCC_THREAD_MULTIPLE means that it supports FUNNELED and MULTIPLE and SINGLE. In the intial version, I had bitmap. @Sergei-Lebedev had pointed this out in the earlier version.

Reply:
Ok, i agree. I just a had a supported modes mask in CL. Yes, since all those values include one anotehr single->funneled->multiple i agree we can use just enum.